### ISSUE TITLE
Inst logo

### ISSUE THREAD

**Issue by Caroline-A** [SDGforskning/find-sdg-research#42]
Er det mulig å bytte ut institusjonenes logo som er brukt nå for versjonene som inneholder institusjonsnavn også? Som søk og skriv.

**Comment by tarjelavik** [comment]
Ja, det er mulig. Jeg tok bare ikoner, for å unngå logikk rundt språk. Footer må også redesignes for å få det til å passe greit sammen.

**Comment by Caroline-A** [comment]
Angående footer, vi bør også ha en tekst som sier “Bærekraftsforskning.no supports the Sustainable Development Goals” tror jeg, siden the colour wheel er brukt ved siden av the header (side 4 https://www.un.org/sustainabledevelopment/wp-content/uploads/2019/01/SDG_Guidelines_AUG_2019_Final.pdf)
